autoload/fern/scheme/file/mapping.vim

```diff
@@ -205,6 +207,28 @@ function! s:map_remove(helper) abort
         \.then({ -> a:helper.sync.echo(printf('%d items are removed', len(ps))) })
 endfunction
 
+function! s:map_preview(helper, mode) abort
+  let node = a:helper.sync.get_cursor_node()
+  if node is# v:null
+    return s:Promise.reject('cursor node is not visible')
+  endif
+  if node.status isnot# a:helper.STATUS_NONE
+    return s:Promise.reject('cannot preview a directory')
+  endif
+
+  try
+    execute "noautocmd bo pedit " .. node._path
```

Comment:
Please make it configurable so that we can define the followings to control the preview window location.

- `fern-action-preview:left`
- `fern-action-preview:right`
- `fern-action-preview:top`
- `fern-action-preview:bottom`
- `fern-action-preview` aliases to `fern-action-preview:bottom` (or one of the aboves)

Reply:
Please use `fnameescape` to escape `node._path` like

```vim
execute printf('noautocmd %s pedit %s', a:prefix, fnameescape(node._path))
```

Reply:
Good suggestions, will do :-)